src/pdf.js

```diff
@@ -42,10 +42,8 @@ if (!PDFJS.workerSrc && typeof document !== 'undefined') {
   // workerSrc is not set -- using last script url to define default location
```

Comment:
This script runs on pdf.js inclusion, means user will not have ability to run his scripts before this one. Logic has to be more complicated to provide this warning and rather at worker construction.